includes/class-object-cache.php

```
@@ -240,6 +240,13 @@ public function enable( $config ) {
 
 		$config_data = $config;
 
+		// Remove password from config file — sensitive credentials are stored
+		// in the `wppo_settings` option (database) and merged at load time via
+		// the `WPPO_REDIS_PASSWORD` constant, environment variable, or the
+		// object cache's REST handler.
+		$password = $config_data['password'] ?? '';
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$password'. (undefined)

(UnusedLocalVariable)